apps/editing-toolkit/editing-toolkit-plugin/wpcom-domain-upsell-callout/src/domain-upsell-callout.scss

```diff
@@ -0,0 +1,70 @@
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+@import "@automattic/color-studio/dist/color-variables";
+
+.wpcom-domain-upsell-callout {
+	font-size: $font-body-extra-small;
+	font-style: normal;
+	padding: 15px;
+	line-height: normal;
+	position: absolute;
+	display: flex;
+	align-items: center;
+	left: 50%;
+	transform: translateX(-50%);
+	top: 0;
+	@media ( max-width: $break-xlarge ) {
+		display: none;
+	}
```

Comment:
There's some clipping going on with the right hand buttons and the upsell. With English it's just a 10 or 20 pixels overlap, with Spanish is quite a bit more. Might consider adjust the `left: 50%` and or adjusting the `max-width: $break-xlarge` to a bigger pixel width.

![clipping](https://github.com/Automattic/wp-calypso/assets/140841/2ce1e5db-40fc-4c5d-b824-25df95c07d06)